include/System/ParamInst.hpp

```diff
@@ -1,13 +1,35 @@
 #ifndef PARAM_INST_HPP
 #define PARAM_INST_HPP
 
+#include "JSystem/JDrama/JDRNameRef.hpp"
 #include "System/BaseParam.hpp"
```

Comment:
For most of the repo I've used the following convention, so lets try to continue it to make stuff consistent:

- `<stuff.h>` is for absolute paths (relative to the `include` folder)
- `"stuff.h"` is for relative paths (relative to the current .cpp file)

And only the `<stuff.h>` is allowed to be used in headers which are inside of `include`, as relative paths would break transitive includes